Website/Providers/DataProviders/SqlDataProvider/07.04.00.SqlDataProvider

```diff
@@ -691,7 +695,7 @@ BEGIN
         'Reject',
         'Reject',
         NULL,
-        1,
+        2,
```

Comment:
Order should be unique per notification type (see all other items). 
I'll submit a pull request for 07.04.01 to modify these values and add a unique key